src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -36,7 +36,7 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
         /// <summary>
         /// Global Nullable context options.
         /// </summary>
-        public NullableContextOptions NullableContextOptions { get; private set; }
+        public override NullableContextOptions NullableContextOptions { get; protected set; }
```

Comment:
>override [](start = 15, length = 8)

can we remove this override?

Reply:
No, the base class version is `abstract`

---
In reply to: [303187489](https://github.com/dotnet/roslyn/pull/37017#discussion_r303187489) [](ancestors = 303187489)